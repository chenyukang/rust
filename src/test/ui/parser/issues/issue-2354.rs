fn foo() { //~ NOTE unclosed delimiter
  match Some(10) {
      Some(y) => { panic!(); }
      None => { panic!(); }
}

fn bar() {
    let mut i = 0;
    while (i < 1000) {}
}

fn main() {}
//~ ERROR this file contains an unclosed delimiter
