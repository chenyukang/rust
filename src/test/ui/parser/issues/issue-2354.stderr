error: this file contains an unclosed delimiter
  --> $DIR/issue-2354.rs:13:52
   |
LL | fn foo() {
   |          - unclosed delimiter
...
LL |
   |                                                    ^

error: aborting due to previous error

